src/app/config.rs

```diff
@@ -546,8 +546,10 @@ impl Strings for Env {
 
 impl TestEnv {
     pub fn new() -> Self {
+        let mut values: HashMap<&'static str, String> = Default::default();
```

Comment:
I don't think this type annotation is strictly necessary.

Reply:
it seems like it is

```
error[E0282]: type annotations needed
   --> src/app/config.rs:550:9
    |
549 |         let mut values = Default::default();
    |             ---------- consider giving `values` a type
550 |         values.insert(ENV_PROXY_ID, "foo.deployment.default.linkerd-managed.linkerd.svc.cluster.local".into());
    |         ^^^^^^ cannot infer type
    |
    = note: type must be known at this point
```

Reply:
It can be `let mut values = HashMap::new()`. It can infer key and value types.